dataplane/dataplane.cpp

```diff
@@ -1024,6 +1165,26 @@ eResult cDataPlane::initQueues()
 	return eResult::success;
 }
 
+eResult cDataPlane::initKniQueues()
+{
+	for (auto& it : kni_interface_handles)
+	{
+		uint16_t port_id = it.first;
+		for (std::size_t i = 0, max = 1; i < max; ++i)
+		{
+			const auto& socket_id = rte_eth_dev_socket_id(port_id);
+			KNIAddTxQueue(i, socket_id);
+		}
+
+		for (std::size_t i = 0, max = 1; i < max; ++i)
```

Comment:
`max = 1` seems strange. I think we should define some constexpr value and name it accordingly